website/sidebars.js

```diff
@@ -43,7 +43,6 @@ const sidebarSettings = {
               ],
             },
             "docs/get-started/dbt-cloud-features",
-            "docs/get-started/connect-your-database",
```

Comment:
@mirnawong1 : just double-checking, was removing this from the leftbar intentional?